docs/devguide/introduction.rst

```diff
@@ -276,6 +276,70 @@ OpenDDS supports the following building blocks, with notes/caveats listed below
 
   * The rest of the building block is not supported.
 
+*********************************************
+Building and Configuring for Interoperability
+*********************************************
+
+The two components needed for interoperability are the :ref:`RTPS Discovery library <rtps-disc>` and the :ref:`RTPS/UDP transport library <rtps-udp-transport>`.
+Both of these libraries are built and installed by default.
+Those using a custom build system need to make sure that these libraries are available to the application.
+
+As of OpenDDS 3, these components will not be used by default and must be configured.
+This can be done via :ref:`environment variables, command-line options, configuration files, and/or code <config-store-keys>`.
+
+The following configuration file snippet changes the default discovery to RTPS and the default transport to RTPS/UDP.
+
+.. code-block:: ini
+
+    [common]
+    DCPSDefaultDiscovery=DEFAULT_RTPS
+    DCPSGlobalTransportConfig=$file
+
+    [transport/the_rtps_transport]
+    transport_type=rtps_udp
+
+The following code accomplishes the same:
+
+.. code-block:: cpp
+
+    using namespace OpenDDS::DCPS;
+    using namespace OpenDDS::RTPS;
+    TransportConfig_rch config =
+      TransportRegistry::instance()->create_config("rtps_config");
+    TransportInst_rch inst =
+      TransportRegistry::instance()->create_inst("the_rtps_transport", "rtps_udp");
+    config->instances_.push_back(inst);
+    TransportRegistry::instance()->global_config(config);
+
+    RtpsDiscovery_rch disc = make_rch<RtpsDiscovery>("RtpsDiscovery");
+    disc->use_xtypes(RtpsDiscoveryConfig::XTYPES_NONE);
```

Comment:
The default value of `use_xtypes` is minimal (see `bool RtpsDiscoveryConfig::use_xtypes() const`), so this should change to match what the ini does.  Also the devguide page that lists these parameters may need to be updated.

Reply:
Should we 1) remove `use_xtypes` from the example since minimal is the default or 2) expand the example with `UseXTypes` to suggest how changing to complete might be done?

Reply:
I think we should go with `2)`, it's a good starting point for the audience of this section of the docs.

Reply:
@jrw972 looks like this one is not resolved?